package.json

```diff
@@ -22,7 +22,7 @@
     "gulp-if": "1.2.5",
     "gulp-plumber": "^1.1.0",
     "gulp-postcss": "^6.4.0",
-    "gulp-sass": "3.1.0",
+    "gulp-sass": "4.1.0",
```

Comment:
This was required because the node-sass packages was not building but then I was testing with latest node.js. Will revert this changes and see if this still builds.